nix/NIX.md

```diff
@@ -1,31 +1,70 @@
-# How to import
+# Running on Nix
 
-To import with flakes use
+## Putting it in your system configuration
+
+### On flakes-enabled nix
+
+#### Directly installing
+
+The `prismlauncher` flake provides a package which you can install along with
+the rest of your packages
 
 ```nix
+# In your flake.nix:
 {
   inputs = {
     prismlauncher.url = "github:PrismLauncher/PrismLauncher";
   };
+}
+```
 
-...
+```nix
+# And in your system configuration:
+environment.systemPackages = [ prismlauncher.packages.${pkgs.system}.prismlauncher ];
 
-  nixpkgs.overlays = [ inputs.prismlauncher.overlay ]; ## Within configuration.nix
-  environment.systemPackages = with pkgs; [ prismlauncher ]; ##
+# Or in your home-manager configuration:
+home.packages = [ prismlauncher.packages.${pkgs.system}.prismlauncher ];
```

Comment:
This is great, much better than encouraging overlays by default in a flakes context.